google-cloud-clients/google-cloud-firestore/src/main/java/com/google/cloud/firestore/Firestore.java

```diff
@@ -117,6 +119,22 @@
   ApiFuture<List<DocumentSnapshot>> getAll(
       @Nonnull DocumentReference[] documentReferences, @Nullable FieldMask fieldMask);
 
+  /**
+   * Retrieves multiple documents from Firestore while optionally applying a field mask to reduce
+   * the amount of data transmitted. Returned documents will be out of order and missing documents
+   * will not be returned.
+   *
+   * @param documentReferences Array with Document References to fetch.
+   * @param fieldMask If not null, specifies the subset of fields to return.
+   * @param transactionId Id of {@link Transaction}
```

Comment:
Ideally, the transaction ID handling would not be exposed in the public API. We currently don't use the transaction ID in any of our customer facing APIs, but instead have helper methods that use them internally.

Reply:
done